libs/langchain-openai/src/azure/embeddings.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  type ClientOptions,
+  AzureOpenAI as AzureOpenAIClient,
+  OpenAI as OpenAIClient,
+} from "openai";
+import { OpenAIEmbeddings, OpenAIEmbeddingsParams } from "../embeddings.js";
+import {
+  AzureOpenAIInput,
+  OpenAICoreRequestOptions,
+  LegacyOpenAIInput,
+} from "../types.js";
+import { getEndpoint, OpenAIEndpointConfig } from "../utils/azure.js";
+import { wrapOpenAIClientError } from "../utils/openai.js";
+
+export class AzureOpenAIEmbeddings extends OpenAIEmbeddings {
+  constructor(
+    fields?: Partial<OpenAIEmbeddingsParams> &
+      Partial<AzureOpenAIInput> & {
+        verbose?: boolean;
+        /**
+         * The OpenAI API key to use.
+         * Alias for `apiKey`.
+         */
+        openAIApiKey?: string;
```

Comment:
This param is only in other integrations for legacy purposes. We shouldn't include in new integrations
```suggestion
```